yetanotherco/aligned_layer
# feat: add operator --version flag

# Description

- This PR adds the `--version` flag to the operator binary.

# To Test

- Run `make build_operator`.
- Run `./operator/build/aligned-operator --version`.
- It should display: `Aligned Layer Node Operator version v0.1.6`.
- To change the value of the version, change `OPERATOR_VERSION` at the top of the makefile. This should be done before each release of the operator to the desired binary version.
